### PR TITLE
[libgeotiff] Update to 1.7.0, revise features and config

### PR DESCRIPTION
**Pull request by dg0yt**
- #### What does your PR fix?  
  Updates and revise libgeotiff.
  Restores canonical `find_package(GeoTiff)` and adds proper `usage`.
  Moves tools to explicit feature.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  all, yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  yes

### CODE REVIEW

File: ports/pdal/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "pdal",
   "version-string": "1.7.1",
```
Comment (JonLiu1993) [inline ports/pdal/vcpkg.json:3]:
```suggestion
  "version": "1.7.1",
```

File: ports/liblas/vcpkg.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "liblas",
   "version-string": "1.8.1",
```
Comment (JonLiu1993) [inline ports/liblas/vcpkg.json:3]:
```suggestion
  "version": "1.8.1",
```

File: ports/libgeotiff/vcpkg.json [hunk 3]
```diff
@@ -1,13 +1,29 @@
 {
   "name": "libgeotiff",
-  "version-string": "1.6.0",
-  "port-version": 4,
-  "description": "Libgeotiff is an open source library normally hosted on top of libtiff for reading, and writing GeoTIFF information tags.",
+  "version-string": "1.7.0",
```
Comment (JonLiu1993) [inline ports/libgeotiff/vcpkg.json:3]:
```suggestion
  "version": "1.7.0",
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Comment by dg0yt** [comment]
@NancyLi1013 Switched to `version` field.

**Comment by dg0yt** [comment]
Now really switched all three fields...

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JonLiu1993** [comment]
All features are tested successfully in the following triplet:
- x86-windows
- x64-windows
- x64-windows-static
- x64-linux

**Pull request merged**